README.md

```diff
@@ -4,26 +4,42 @@
 
 🧱 A composable solution for building reusable lists in Swift/UIKit. As easy as playing with Legos.
 
-- [Features](#features)
-- [Installation](#installation)
-    - [Swift Package Manager](#swift-package-manager)
-- [Contributing](#contributing)
-- [Authors](#authors)
-
-# Features
+- [BricksAndTiles](#bricksandtiles)
+  - [Features](#features)
+    - [UITableView](#uitableview)
+    - [UICollectionView](#uicollectionview)
+    - [UITableView + UICollectionView](#uitableview--uicollectionview)
+  - [Installation](#installation)
+      - [Swift Package Manager](#swift-package-manager)
+  - [Contributing](#contributing)
+  - [Authors](#authors)
+
+## Features
 ### UITableView
 - [x] Static cells
 - [x] Draggable cells
+
+![](demoAssets/draggable.gif)
+
 - [x] Cells with actions on slide
+
+![](demoAssets/slide_actions.gif)
+
 - [x] Single selection
 
+![](demoAssets/selectable.gif)
+
 ### UICollectionView
 - [x] Static collection cells
 
+![](demoAssets/collection.gif)
+
```

Comment:
We got a PR of a Grid collection last week, what do you think of adding that feature here as well?

Reply:
Ah yes, I made one and put it in the `./demoAssets` but I must've forgotten to add it. I'll get right on that.